janusgraph-hadoop-parent/janusgraph-hadoop-core/src/main/java/org/janusgraph/hadoop/config/job/AbstractDistCacheConfigurer.java

```diff
@@ -146,7 +146,7 @@ private static Conf configureByClasspath(String mapredJarFilename) {
         for (String cpentry : classpath.split(File.pathSeparator)) {
             if (cpentry.toLowerCase().endsWith(".jar") || cpentry.toLowerCase().endsWith(".properties")) {
                 paths.add(new Path(cpentry));
-                if (cpentry.toLowerCase().endsWith(mrj));
+                if (cpentry.toLowerCase().endsWith(mrj))
```

Comment:
This is a behavior change — while it's most likely correct and the `;` was a typo, it will now do assignments conditionally, whereas before, it would be unconditional.

Does this have potential to break some MapReduce workflows as a result? Should it be documented in release notes or otherwise that this is, while a correct change, is a potential issue to watch out for when upgrading from Titan to JanusGraph or from previous JanusGraph release to later ones?